lib/systemd_testsuite_test.pm

```diff
@@ -20,17 +20,26 @@ use known_bugs;
 use testapi;
 use power_action_utils 'power_action';
 use utils 'zypper_call';
-use version_utils;
+use version_utils qw(is_opensuse is_sle is_tumbleweed);
 
 
 sub testsuiteinstall {
     my $qa_head_repo = get_var('QA_HEAD_REPO', '');
```

Comment:
I know it's not your change, but, if QA_HEAD_REPO is required, could you change it to get_required_var and adapt the code? Would save one level of identation :dancer:

Reply:
It is not required, it is optional to be able to override the repository with a home OBS repo when cloning.

Reply:
Yeah, but look at Line 44. Basically means if this function is called, and variable is not set, test dies... That was my point :P but no probs, it aint your code anyway (I think)

Reply:
Yes, because it is required that the variable exists, but not the isotovideo SETTING.

Reply:
Got it, one might want to override this setting.